debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java

```diff
@@ -1086,13 +1086,6 @@ public static AutoCreateMode parse(String value, String defaultValue) {
     public static final Field SOURCE_INFO_STRUCT_MAKER = CommonConnectorConfig.SOURCE_INFO_STRUCT_MAKER
             .withDefault(PostgresSourceInfoStructMaker.class.getName());
 
-    public static final Field TASK_ID = Field.create("task.id")
```

Comment:
This was removed since we already have a `task.id` field in the `CommonConnectorConfig` of type `STRING` - we are now reusing the same config.